Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
@MikeMcQuaid Are you doing some check? 
By the way, as a linux user I am in love with HB since with this installer system I can use my Mac mostly as a linux user. In a general purpose, if I want to share a binary  do you give me the advice to use macports or some another installer system accepting both binary and source-based installation. I thought it was not a good idea to mix different installer systems. I'll try once and I never do it again. I thought also Homebrew cask was the right tool to install a binary. So at some point, I am a bit lost ... I understand your point to prefer source-based installation but I do not understand why you do not want to accept to share it as a binary too. It is like apt-get only does one kind of installation (binary or from source) and not both at the same time. As an example, R is distributed as a cask and maybe not as a Formula (if I am not mistaken) but R can be installed from source on MacOSX. So I am lost ... sorry ...